src/pickup.cpp

```diff
@@ -481,14 +564,15 @@ void Pickup::autopickup( const tripoint &p )
             }
         }
     }
-
-    std::vector<item_location> target_items;
     std::vector<int> quantities;
+    // create quantities for already registered target items to pickup
+    for( size_t i = 0; i < target_items.size(); i++ ) {
+        quantities.push_back( 0 );
+    }
```

Comment:
```suggestion
    std::vector<int> quantities( target_items.size(), 0 );
```

Reply:
Resolved in commit 7b670045403370fc9b7132711d2dc715a1422270.